scripts/example/jupyterlab_version/l8_cloud_dataset.yaml

```diff
@@ -0,0 +1,37 @@
+dataset:
+  images:
+    type: tiff
+    extension: _data.tif
+    directory: train
+  labels:
+    extension: _mask.png
+    type: tiff
+    directory: labels
+  classes:
```

Comment:
instead of duplicating these files, could we get rid of jupyterlab_version folder, put everything in the example folder, and use the same helper files for both versions?